### ISSUE TITLE
BTC/CNY Bisq Liquidity Week

### ISSUE THREAD

**Issue by ripcurlx** [bisq-network/growth#45]
## Learning Goal/Objective
Provide enough liquidity for the BTC/CNY order book to bootstrap the market for organic growth. Lasting 1 week from **TBD**.

Objective: 5 traders placing 10 offers on the BTC/CNY order book for a week.

## Hypothesis
If the experiment is successful, we predict to the total volume to grow by 30% compared to the previous week.

## Experiment Design
Every participant on the experiment tries to keep two offers (1 buy, 1 sell ) open all the time to provide liquidity.

## Bounty
Participants in the experiment who meet the above requirement (creating at least 2 offers during the week) will be rewarded with **20 $BSQ**.

## Resource Estimation and Probability
Low personal effort and High probability that it will succeed.

## Results
> _- What happened?_

## Learnings
> _-What did we learn by running this experiment?_

## Action Items
> _- Another experiment for this goal?_
> _- How do we progress from here on?_

**Comment by flix1** [comment]
Since bounty #135 is currently ongoing, it might be a good idea to coordinate these two. Maybe we could make it coincide with the last week of that month. Or maybe the first week after that bounty is completed... Any ideas @RadishXN ?

For reference see previous liquidity weeks here: 
#12      
#36

**Comment by RadishXN** [comment]
> Since bounty #135 is currently ongoing, it might be a good idea to coordinate these two. Maybe we could make it coincide with the last week of that month. Or maybe the first week after that bounty is completed... Any ideas @RadishXN ?
> 
> For reference see previous liquidity weeks here:
> #12
> #36

Yes, I am happy to. But form now,I think I am going to need a bit more time. Anyway, I will give my idea tomorrow. As for the time, I thought last week of that months will be a perfect time. And I am sorry that I am so busy today. I have to get you the answer tomorrow. sorry again.
